### ISSUE TITLE
PassphraseDialogActivity.java:203 NullPointerException

### ISSUE THREAD

**Issue by dschuermann** [open-keychain/open-keychain#1672]
From Google Play dev console.

3.8
General Mobile 4G (gm4g_s_sprout)
"Crashed when was trying decryption."

```
java.lang.RuntimeException: Unable to resume activity {org.sufficientlysecure.keychain/org.sufficientlysecure.keychain.ui.PassphraseDialogActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'long java.lang.Long.longValue()' on a null object reference
	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3103)
	at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3134)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2481)
	at android.app.ActivityThread.-wrap11(ActivityThread.java)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:148)
	at android.app.ActivityThread.main(ActivityThread.java:5417)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'long java.lang.Long.longValue()' on a null object reference
	at org.sufficientlysecure.keychain.ui.PassphraseDialogActivity$PassphraseDialogFragment.onCreateDialog(PassphraseDialogActivity.java:203)
	at android.support.v4.app.DialogFragment.getLayoutInflater(DialogFragment.java:308)
	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1067)
	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1248)
	at android.support.v4.app.BackStackRecord.run(BackStackRecord.java:738)
	at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:1613)
	at android.support.v4.app.FragmentController.execPendingActions(FragmentController.java:330)
	at android.support.v4.app.FragmentActivity.onPostResume(FragmentActivity.java:452)
	at android.app.Activity.performResume(Activity.java:6351)
	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3092)
	... 10 more
```

**Comment by Valodim** [comment]
symmetric passphrase decryption is broken - this one's on me. derp.

**Comment by dschuermann** [comment]
np, I released 3.8.1
